src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -201,15 +193,17 @@ static RuntimeMetrics()
             exceptionCounter.Add(1);
         };
     }
-
+#pragma warning disable SA1313
     /// <summary>
     /// Initializes a new instance of the <see cref="RuntimeMetrics"/> class.
     /// </summary>
-    /// <param name="options">The options to define the metrics.</param>
-    public RuntimeMetrics(RuntimeInstrumentationOptions options)
+    /// <param name="_1">The options to define the metrics.</param>
+    public RuntimeMetrics(RuntimeInstrumentationOptions _1)
+#pragma warning restore SA1313
```

Comment:
A bit odd. Why not just remove the parameter?

```diff
-public RuntimeMetrics(RuntimeInstrumentationOptions _1)
+public RuntimeMetrics()
```

Could make the whole class static. But need some way to make sure it is bootstrapped. 

Like instead of...

```csharp
        var instrumentation = new RuntimeMetrics(options);
        builder.AddMeter(RuntimeMetrics.MeterInstance.Name);
        return builder.AddInstrumentation(() => instrumentation);
```

Do something like...

```csharp
        GC.KeepAlive(RuntimeMetrics.SomethingAccessibleToTriggetStaticCtor)
        return builder.AddMeter(RuntimeMetrics.MeterInstance.Name);
```

Reply:
Created #2289

The problem is with public contract if extension method. We need to keep this as the package is released as stable. Then we need to suppress warning there. I am not against, but lets do it seprately

```
    public static MeterProviderBuilder AddRuntimeInstrumentation(
        this MeterProviderBuilder builder,
        Action<RuntimeInstrumentationOptions>? configure)
```